energy_analyse2020.10.16_213101.R

```diff
@@ -0,0 +1,177 @@
+#set path of data and get all the subforders
+#change the path where you put the data
+setwd("D:/Rdata/2020.10.16_213101/data/nexus6p") 

```

Comment:
Using relative paths would be preferable (e.g. start with `/data/...` where data is the folder in the repo where the data is stored)

Reply:
sure, will change it in next version(I just abstract the common path instead)